test/e2e/lib/gutenberg/blocks/index.js

```diff
@@ -0,0 +1,18 @@
+// TODO some blocks are exported as default and some not... maybe change all of them to export as default?
```

Comment:
Consistency is likely the best approach. Maybe in another PR we change them all to use one or the other.

Slight preference from me for named exports here.